website/public/cli/index.html

```diff
@@ -3,444 +3,343 @@
 <!-- See end of file for extended copyright information. -->
 <!-- This file was generated using generate-cli-docs. -->
 <html lang="en">
-  <head>
```

Comment:
Run `cd website && yarn && yarn fmt` to reformat this code and minimize the diff.

Reply:
thats lemme that i can't run that with debian sid nodejs version (12.something), it's been 1 hour compiling node, done.